packages/office-ui-fabric-react/src/components/SwatchColorPicker/ColorPickerGridCell.styles.ts

```diff
@@ -22,6 +22,11 @@ export const getStyles = (props: IColorPickerGridCellStyleProps): IColorPickerGr
   const { theme, disabled, selected, circle, isWhite, height = DEFAULT_CELL_SIZE, width = DEFAULT_CELL_SIZE, borderWidth } = props;
   const { semanticColors } = theme;
 
+  const buttonBorderHovered = theme.palette.neutralLighter;
+  const buttonBorderChecked = theme.palette.neutralLight;
+  const buttonBorderCheckedHovered = theme.palette.neutralSecondary;
+  const buttonBorderIsWhite = theme.palette.neutralTertiary;
```

Comment:
nit: Why not de-structure the palette? Also, do we not have semantic slots for it? Aren't they supposed to be similar to a button?

Reply:
Was speaking with Makoto and he is making changes to the buttonBorder semanticColors so I'll just keep my changes pointed at palette until his changes are in and then just switch these over.